src/components/Hello.tsx

```diff
@@ -21,14 +23,16 @@ export default function Hello({ compiler, framework }: Props): JSX.Element {
   return (
     <form onSubmit={handleSubmit}>
       <h1>{t('Hello', { name: name || 'World', framework, compiler })}</h1>
-      <label htmlFor="name">{t('Name')}</label>
-      <input
-        id="name"
-        type="text"
-        value={nameField}
-        onChange={(e) => setNameField(e.target.value)}
-      />
-      <input type="submit" value="Submit" />
+      <div className={classes.form}>
+        <label htmlFor="name">{t('Name')}</label>
```

Comment:
Eventually, you will want to change these to Material inputs. It's quite instructive doing so. Material is definitely verbose!

Reply:
Oh yes, I am still getting used to it!